compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -922,7 +922,8 @@
                 },
                 {
                     "name": "paramName",
-                    "type": "Token"
+                    "type": "Token",
+                    "isOptional" : true
```

Comment:
Param name for other two should be optional as well?

Reply:
Yes, they are as well. Didn't fix it since it wasn't related to this test. Is it better to fix it as well?

Reply:
+1 lets fix it. Otherwise we may miss it.

Reply:
fixed with 09f7f33